tool/common_mark_stats.txt

```diff
@@ -18,11 +18,11 @@
   22 of   26 –  84.6%  Lists
    8 of    8 – 100.0%  Paragraphs
    1 of    1 – 100.0%  Precedence
-  16 of   21 –  76.2%  Raw HTML
+  21 of   21 – 100.0%  Raw HTML
   26 of   27 –  96.3%  Setext headings
    2 of    2 – 100.0%  Soft line breaks
   11 of   11 – 100.0%  Tabs
    3 of    3 – 100.0%  Textual content
   19 of   19 – 100.0%  Thematic breaks
- 615 of  652 –  94.3%  TOTAL
- 549 of  615 –  89.3%  TOTAL Strict
```

Comment:
What's with the "TOTAL Strict" decreasing by 0.1%? With a few more decimals, it looks like we moved from 89.27% to 89.19%? But everything got strictly better? Maybe you need to regenerate once more?

Reply:
I was confused too 😄  Because the strict is based on total which contains the loose ones.

Reply:
You could see the total has increases by 5(from 615 to 620), and the the total strict has increased by 4(from 549 to 553)

Reply:
so weird. A problem for another day!